### PR TITLE
LPS-175478

### PR DESCRIPTION
**Pull request by marcosvictor22**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by marcosvictor22** [comment]
ci:test:sf

**Comment by marcosvictor22** [comment]
@david-gutierrez-mesa @Tim-Cao

**Comment by marcosvictor22** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 1]
```diff
@@ -2393,6 +2393,55 @@ definition {
...
+				groupName = "Global",
+				title = "Web Content Title");
+
+			HeadlessSite.addSite(siteName = "Test Site Name");
+		}
+
+		task ("Given that a user has enabled Local Live on the Site") {
+			Staging.openStagingAdmin(siteURLKey = "Test Site Name");
+
+			Staging.activateStagingCP(siteName = "Test Site Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:2411]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 2]
```diff
@@ -2393,6 +2393,55 @@ definition {
...
+		}
+
+		task ("When the user creates other Web Content") {
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title 1");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:2418]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 3]
```diff
@@ -2393,6 +2393,55 @@ definition {
...
+
+			Navigator.gotoSitePage(
+				pageName = "Test Site Name",
+				siteName = "Test Site Name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:2422]:
```
			JSONLayout.addPublicLayout(
				groupName = "${randomSiteName} (Staging)",
				layoutName = "Test Page Name",
				site = "false",
				type = "content");
				

			ContentPagesNavigator.openEditContentPage(
				pageName = "Test Page Name",
				siteName = "${randomSiteName} Staging");

			PageEditor.addWidget(portletName = "Web Content Display");

		PageEditor.addWidget(
			portletName = "Web Content Display",
			targetFragmentName = "Asset Publisher");
				
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 4]
```diff
@@ -2393,6 +2393,55 @@ definition {
...
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Site Name",
+				site = "false",
+				widgetName = "Web Content Display");
+
+			PageEditor.addWidget(
+				dropTarget = "Header",
+				portletName = "Asset Publisher",
+				scope = "Global");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:2435]: Should be these steps

8. Configure the Web Content Display widget to have the Global Scope (under the "Scope" tab), then configure it to display the "Global Article" from step 2. Click Save.
9. Beneath the Web Content Display widget, add an Asset Publisher widget.
10. Configure the Asset Publisher widget as follows:

Asset Selection: Manual

Scope: Click "Select", then "Global". The Scope should now be showing as "Current Site (Test Site)" + "Global"

Asset Entries: Basic Web Content > Site Article.

Save

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase [hunk 5]
```diff
@@ -2393,6 +2393,55 @@ definition {
...
+		}
+
+		task ("Then Asset Publisher would display the Site Article") {
+			Staging.openStagingAdmin(siteURLKey = "${randomSiteURLKey}-staging");
+
+			Staging.publishCustomPublication();
+		}
+	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase:2443]: Miss steps please check the ticket https://liferay.atlassian.net/browse/LPS-153676

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @marcosvictor22, there are some suggestions when you write automation.

1. Manual Testing follows the steps in bug ticket.
2. Make sure you fully understand the testing point.
3. Run test after write the automation.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase (hunk 4 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase]: Hi @marcosvictor22, could you have a look at this comment?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase (hunk 5 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/WebContent.testcase]: Also this one @marcosvictor22

**Comment by Tim-Cao** [comment]
See changes @marcosvictor22
https://github.com/liferay-echo/liferay-portal/pull/13283

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**